### DISCUSSION TITLE
[FEATURE] waf需要怎么设置 屏蔽大量不同恶意IP 访问网站

### DISCUSSION THREAD

**Discussion by Alexzzz2008** [1Panel-dev/1Panel#6258] [Q&A]
### 1Panel 版本

1.10.13

### 请描述您的需求或者改进建议

网站被大量不同的恶意IP，一直在访问网站首页，造成网络堵塞，CPU负荷占满，

WAF应该怎么设置 才能屏蔽这些恶意请求，并且不影响正常访问

### 请描述你建议的实现方案

1

**Comment by zhengkunwang223** [comment]
1. 开启 CC 防护
2. 专业版可以针对首页开启人机验证

**Reply by Alexzzz2008** [reply]
CC防护在哪设置  具体怎么设置

**Reply by zhengkunwang223** [reply]
> CC防护在哪设置 具体怎么设置

高级功能-WAF-网站设置

**Reply by Alexzzz2008** [reply]
能不能协助解决，可以再买一份专业版

**Reply by zhengkunwang223** [reply]
> 能不能协助解决，可以再买一份专业版

![image](https://github.com/user-attachments/assets/191deb6c-0873-44b9-9ed8-aa5c0800cf8c)
加我企业微信
